### PR TITLE
Check for which user is runing the script.  If running as root, assum…

### PR DESCRIPTION
**Pull request by tonybaltovski**
…e user is administrator.  This ensures certain commands are ran as the user rather than root.

### CODE REVIEW

File: install.sh [hunk 1]
```diff
@@ -114,124 +193,303 @@ else
...
 else
-  echo -e "\e[94mCreating default robot YAML \e[0m"
-  sudo cp /opt/ros/humble/share/clearpath_config/sample/a200_default.yaml /etc/clearpath/robot.yaml
-  # Check if sources were added
-  if [ ! -e /etc/clearpath/robot.yaml ]; then
-    echo -e "\e[31mError: Clearpath robot YAML, exiting\e[0m"
-    exit 0
-  fi
+    echo "Service file /etc/systemd/system/network-online.target.wants/systemd-networkd-wait-online.service not found."
 fi
```
Comment (hilary-luo) [inline install.sh:213]: Missing Done message for the Configuring network service section (was `echo -e "\e[32mDone: Configuring network service\e[0m"`)

File: install.sh [hunk 2]
```diff
@@ -114,124 +193,303 @@ else
...
   sudo rm /etc/needrestart/conf.d/10-auto-cp.conf
 fi
 
-echo -e "\e[94mConfiguring network service, if needed\e[0m"
-# Check if the service file exists
-if [ -e "/etc/systemd/system/network-online.target.wants/systemd-networkd-wait-online.service" ]; then
-    # Check if TimeoutStartSec is present in the service file
-    if grep -q "TimeoutStartSec=2sec" "/etc/systemd/system/network-online.target.wants/systemd-networkd-wait-online.service"; then
-        echo "TimeoutStartSec is already present in /etc/systemd/system/network-online.target.wants/systemd-networkd-wait-online.service"
+echo -e "\e[32mDone: Configuring network service\e[0m"
```
Comment (hilary-luo) [inline install.sh:354]: I believe these are the two lines that belong up at line 213

File: install.sh [hunk 3]
```diff
@@ -114,124 +193,303 @@ else
...
+echo ""
+
+### USER SECTION
```
Comment (hilary-luo) [inline install.sh:357]: Is this a repeat of line 214 - 352?

**Review by hilary-luo — CHANGES_REQUESTED** [review]
I think there may be some unintentional duplication here

File: install.sh [hunk 4]
```diff
@@ -114,124 +193,152 @@ else
   fi
 fi
 
-rosdep -q update
 echo -e "\e[32mDone: Configuring rosdep\e[0m"
 echo ""
 
-
-echo -e "\e[94mConfiguring Clearpath Setup\e[0m"
```
Comment (hilary-luo) [inline install.sh:122]: We still need to check for / create the folder and set up the user rights one way or another, is this functionality being moved elsewhere?

File: install.sh [hunk 5]
```diff
@@ -114,124 +193,152 @@ else
...
 fi
 
-echo -e "\e[32mDone: Configuring network service\e[0m"
-echo ""
 
-echo -e "\e[32mClearpath Computer Installer Complete\e[0m"
-echo -e "\e[94mTo continue installation visit: https://docs.clearpathrobotics.com/docs/ros/networking/computer_setup \e[0m"
-echo -e "\e[94mTo start the robot service run:\e[0m sudo systemctl daemon-reload && sudo systemctl start clearpath-robot.service"
+# Reenable messages about restarting services in systems with needrestart installed
+if [ -d /etc/needrestart/conf.d ]; then
```
Comment (hilary-luo) [inline install.sh:342]: Can we check if the file itself exists, not just the folder?

File: install.sh [hunk 6]
```diff
@@ -74,19 +160,12 @@ fi
...
 echo ""
 
-
 echo -e "\e[94mUpdating packages and installing ROS 2\e[0m"
 sudo apt -y -qq update
 sudo apt install ros-humble-ros-base python3-argcomplete ros-dev-tools python3-vcstool ros-humble-clearpath-robot python3-clearpath-computer-setup -y
 echo -e "\e[32mDone: Updating packages and installing ROS 2\e[0m"
 echo ""
 
-echo -e "\e[94mSetting up enviroment\e[0m"
```
Comment (hilary-luo) [inline install.sh:84]: Is there a reason we don't want to still do this, just under the user section?

**Review by hilary-luo — CHANGES_REQUESTED** [review]

File: install.sh (hunk 6 above)
Comment (tonybaltovski) [reply install.sh:84]: It is below in 288

File: install.sh (hunk 5 above)
Comment (tonybaltovski) [reply install.sh]: Added.

File: install.sh (hunk 4 above)
Comment (tonybaltovski) [reply install.sh:122]: Re-added.

**Review by hilary-luo — APPROVED** [review]
Looks good!

**Pull request merged**